server/grpc/api/service.go

```diff
@@ -11,19 +12,21 @@ import (
 // ServiceServer is the type to aggregate all Service APIs.
 type ServiceServer struct {
 	sdk *sdk.SDK
+	cfg *config.Config
 }
 
 // NewServiceServer creates a new ServiceServer.
-func NewServiceServer(sdk *sdk.SDK) *ServiceServer {
-	return &ServiceServer{sdk: sdk}
+func NewServiceServer(sdk *sdk.SDK, cfg *config.Config) *ServiceServer {
+	return &ServiceServer{sdk: sdk, cfg: cfg}
 }
 
 // Create creates a new service from definition.
 func (s *ServiceServer) Create(ctx context.Context, req *protobuf_api.CreateServiceRequest) (*protobuf_api.CreateServiceResponse, error) {
-	credUsername, credPassphrase, err := GetCredentialFromContext(ctx)
-	if err != nil {
-		return nil, err
-	}
+	// credUsername, credPassphrase, err := GetCredentialFromContext(ctx)
```

Comment:
Is it too complex to have it in the context and we just override the context with an interceptor?
This way we still use that credentials from context and when we migrate to multi-account we just need to remove the interceptor.

This way the serviceserver will not have any knowledge about the config too.